README.Rmd

```diff
@@ -64,3 +64,33 @@ my_cal_id <- "apdevries@gmail.com"
 events <- calendar_events(my_cal_id, days_in_past = 90, days_in_future = 90)
 events
 ```
+
+
+## Creating calendar events
```

Comment:
Added to README to make clear auth requirements for creating events as well as examples to do so.